controllers/selfnoderemediation_controller.go

```diff
@@ -221,6 +225,20 @@ func (r *SelfNodeRemediationReconciler) ReconcileManager(ctx context.Context, re
 		}
 	}()
 
+	if isConfigurationExist, err := r.isConfigurationExist(ctx); err != nil {
+		return ctrl.Result{}, err
+	} else if !isConfigurationExist {
+		r.logger.Info("SNR is disabled because configuration does not exist, waiting for configuration creation ")
+		meta.SetStatusCondition(&snr.Status.Conditions, metav1.Condition{
+			Type:    string(v1alpha1.DisabledConditionType),
+			Status:  metav1.ConditionTrue,
+			Reason:  string(snrDisabledNoConfig),
+			Message: "snr is disabled because configuration does not exist",
```

Comment:
nit: this is a user facing message, IMHO uppercase SNR or even full written SelfNodeRemediation would be better here